src/test/java/calebyyy/ParserTest.java

```diff
@@ -0,0 +1,42 @@
+package calebyyy;
+
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import calebyyy.commands.Command;
+import calebyyy.commands.ListCommand;
+import calebyyy.exceptions.InvalidCommandException;
+
+public class ParserTest {
+    private Parser parser;
+    private TaskList taskList;
+    private Storage storage;
+    private Ui ui;
+    private Calebyyy calebyyy;
+
+    @BeforeEach
+    public void setUp() {
+        ui = new Ui();
+        taskList = new TaskList(ui);
+        storage = new Storage("data/tasks.txt");
+        calebyyy = new Calebyyy();
+        parser = new Parser(calebyyy, taskList, storage, ui);
+    }
+
+    @Test
+    public void testGetCommand_validCommand() {
```

Comment:
Do use the suggested naming for tests: `featureUnderTest_testScenario_expectedBehavior()`
e.g. `sortList_emptyList_exceptionThrown()`, `getMember_memberNotFound_nullReturned`